hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/services/TestApacheClientConnectionPool.java

```diff
@@ -118,14 +161,18 @@ public void testKeepAliveCacheCleanup() throws Exception {
   @Test
   public void testKeepAliveCacheCleanupWithConnections() throws Exception {
     try (KeepAliveCache keepAliveCache = new KeepAliveCache(
-        new AbfsConfiguration(new Configuration(), ""))) {
+        new AbfsConfiguration(new Configuration(), EMPTY_STRING))) {
       keepAliveCache.pauseThread();
       keepAliveCache.clear();
       HttpClientConnection connection = Mockito.mock(
           HttpClientConnection.class);
       keepAliveCache.put(connection);
 
       Thread.sleep(2 * keepAliveCache.getConnectionIdleTTL());
```

Comment:
how long is this sleep? i ask as in #6907 tuning two tests cuts the unit test time of that module down by a minute. I think for this test case the timeout should be milliseconds, not seconds

Reply:
In the tests in which sleep is needed, have reduced TTL to 100 ms, so the thread sleep would be maximum 200 ms.